browser/brave_profile_prefs.cc

```diff
@@ -114,6 +115,11 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(prefs::kImportDialogWindows, true);
   // Importer: ledger (used for Brave Rewards pinned => tips)
   registry->RegisterIntegerPref(kBravePaymentsPinnedItemCount, 0);
+
+  registry->RegisterBooleanPref(
```

Comment:
these should go in `RewardsService::RegisterProfilePrefs`

Reply:
Moved.